addons/account_edi_ubl_cii/models/account_move_send.py

```diff
@@ -34,7 +34,7 @@ def _compute_checkbox_ubl_cii_label(self):
             code_to_label = dict(wizard.move_ids.partner_id._fields['ubl_cii_format'].selection)
             codes = wizard.move_ids.partner_id.mapped('ubl_cii_format')
             if any(codes):
-                wizard.checkbox_ubl_cii_label = ", ".join(code_to_label[c] for c in codes)
+                wizard.checkbox_ubl_cii_label = ", ".join(code_to_label[c] for c in codes if c)
```

Comment:
The fix is good but I think we have another issue when there are multiple invoices linked to != partners having the same `ubl_cii_format`. In that case, we are displaying `"Facturx, Facturx"`. Can you fix it as well plz? :)

Reply:
oh, that's true! didn't notice before